packages/@react-spectrum/s2/style/style-macro.ts

```diff
@@ -11,8 +11,13 @@
  */
 
 import type {Condition, CSSProperties, CSSValue, CustomValue, Property, PropertyValueDefinition, PropertyValueMap, RenderProps, ShorthandProperty, StyleFunction, StyleValue, Theme, ThemeProperties, Value} from './types';
+import fs from 'fs';
 import * as propertyInfo from './properties.json';
 
+// Postfix all class names with version for now.
+const json = JSON.parse(fs.readFileSync(__dirname + '/../package.json', 'utf8'));
+const POSTFIX = json.version.includes('nightly') ? json.version.match(/-nightly-(.*)/)[1] : json.version.replace(/[0.]/g, '');
```

Comment:
3.20.1 === 3.2.1?
=> 321

Reply:
It's 0.8.0, not 3.x. And if it goes to 0.10 that's ok because we never had a 0.1 prefix.

Reply:
true, i guess it'd have to get to 80's

the 3 was just a random number that happened to coincide. it was more that I was suggesting that numbers could conflict